File: python_programming/practices/time.py
# MM 6th Time of Day
import datetime

current = datetime.datetime.now()
hour = current.hour

hour = int(input("What's the time (in military time, plain military time).\n"))

if hour >= 1 and hour <= 11:
    print("Good morning!")
elif hour >= 12 and hour < 19:
    print("Good afternoon!")
else:
    print("Good night!")

File: c_programming/practices/time.c
// MM 6th CSP Time of Day
#include <stdio.h>

int main(void){

    int time;
    printf("What time is it, in military standards?\n");
    scanf("%d", &time);

    if(time >= 1 && time <= 11){
        printf("Good morning!\n");
    }else if(time >= 12 && time < 19 ){
        printf("Good afternoon!\n");
    }else{
        printf("Good night!\n");
    }

    return 0;
}